.circleci/config.yml

```diff
@@ -199,6 +201,12 @@ workflows:
           requires:
             - image-build
           filters: *tagged_build_filters
+      - slack/approval-notification:
+          webhook: ${SLACK_WEBHOOK}
+          message: "Deploy pending approval: https://circleci.com/workflow-run/$CIRCLE_WORKFLOW_ID"
```

Comment:
Can you change the notation from `$CIRCLE_WORKFLOW_ID` to `${CIRCLE_WORKFLOW_ID}` in order to avoid confusion?

Reply:
@smusali please make sure i did this right! thank you so much